### ISSUE TITLE
Uncaught TypeError: s.elementsCensor is not a function

### ISSUE THREAD

**Issue by IsaacHub** [almende/vis#2918]
**The issue with only v4.19.0 and v4.19.1**
While dragging the timeline handle, all browsers throws the following error:
But works perfectly for versions v4.18.1 and below.

```
vis.js:28488 Uncaught TypeError: s.elementsCensor is not a function
    at o._onDrag (vis.js:28488)
    at Array.s (vis.js:13816)
    at Object.emit (vis.js:16351)
    at e (vis.js:15359)
    at nt.emit (vis.js:15364)
    at nt.emit (vis.js:15649)
    at nt.tryEmit (vis.js:15387)
    at nt.recognize (vis.js:15434)
    at Object.recognize (vis.js:16200)
    at W.T [as callback] (vis.js:14349)
o._onDrag @ vis.js:28488
s @ vis.js:13816
emit @ vis.js:16351
e @ vis.js:15359
emit @ vis.js:15364
emit @ vis.js:15649
tryEmit @ vis.js:15387
recognize @ vis.js:15434
recognize @ vis.js:16200
T @ vis.js:14349
handler @ vis.js:14741
M.domHandler @ vis.js:14262
```

By the way the issue with this which I downloaded the code from [here](http://leafletplayback.theoutpost.io/) _(this demo uses v3.x so no issue with this but I want to use latest v4.19.1 which gives error)_ which is used in conjunction with leaflet plugin [source](https://github.com/hallahan/LeafletPlayback)

**Comment by IsaacHub** [comment]
Issue #2884 is related to this issue.

**Comment by mojoaxel** [comment]
see also #2420 and #1906

**Comment by darthsteven** [comment]
Should be fixed in #2947
